test/image/mocks/gl3d_ibm-plot.json

```diff
@@ -304,6 +304,7 @@
         "scene": {
             "xaxis": {
                 "title": "Term",
+                "hovertitle": "Term",
```

Comment:
Adding `hovertitle` to a few mock is ok, but we should really test this issue jasmine tests.

- https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/hover_label_test.js for cartesian 
- https://github.com/plotly/plotly.js/blob/884d2fa4cf76529bd6cb819fa2d4fec1f43fadfa/test/jasmine/tests/ternary_test.js#L120-L174 for ternary 
- https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/gl3d_plot_interact_test.js#L107 for gl3d
- https://github.com/plotly/plotly.js/blob/884d2fa4cf76529bd6cb819fa2d4fec1f43fadfa/test/jasmine/tests/scatterpolar_test.js#L83-L200 for polar
- https://github.com/plotly/plotly.js/blob/884d2fa4cf76529bd6cb819fa2d4fec1f43fadfa/test/jasmine/tests/carpet_test.js#L633-L681 for carpet

----

Oh and we should also `hovertitle` to `geo.lonaxis` and `geo.lataxis` as displayed on https://codepen.io/etpinard/pen/gqKjeB